numba/core/cgutils.py

```diff
@@ -678,7 +678,7 @@ def _dbg():
                        "for axis {} with size %d\n".format(axis), ind, dimlen)
             else:
                 printf(builder, "debug: IndexError: index %d is out of bounds "
-                       "for axis %d with size %d\n".format(axis), ind, axis,
+                       "for axis %d with size %d\n", ind, axis,
```

Comment:
Yeah, this one I am not so sure about...

Reply:
Think it's a copy paste bug. There's nothing to format and the rest of the args are for the `printf`. It's like doing `"hello".format(123)`.

Reply:
Yeah, I think you are correct.